subworkflows/echo-workflow.cwl

```diff
@@ -13,7 +13,7 @@ outputs:
 
 steps:
   echo:
-    run: https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/cwl-tool-template/v0.0.15/cwl/echo.cwl
+    run: https://raw.githubusercontent.com/Sage-Bionetworks-Workflows/cwl-tool-template/main/cwl/echo.cwl
```

Comment:
This should always use a version number for the sake of stability.

Reply:
good call, fixed